File: visualBasic/Ice/minimal/HelloI.vb
' **********************************************************************
'
' Copyright (c) 2003-2014 ZeroC, Inc. All rights reserved.
'
' **********************************************************************

Imports Demo

Public NotInheritable Class HelloI
    Inherits HelloDisp_

    Public Overloads Overrides sub sayHello(ByVal current As Ice.Current)
        System.Console.Out.WriteLine("Hello World!")
    End Sub
End Class

File: java-compat/IceBox/hello/HelloI.java
// **********************************************************************
//
// Copyright (c) 2003-present ZeroC, Inc. All rights reserved.
//
// **********************************************************************

import Demo.*;

public class HelloI extends _HelloDisp
{
    @Override
    public void
    sayHello(Ice.Current current)
    {
        System.out.println("Hello World!");
    }
}
